packages/calypso-products/src/types.ts

```diff
@@ -18,9 +18,12 @@ import type {
 	JETPACK_RESET_PLANS,
 	TERMS_LIST,
 	TYPES_LIST,
+	PERIOD_LIST,
 } from './constants';
 
-export type Features = keyof typeof features;
+const featureValues = Object.values( features );
+
+export type Feature = typeof featureValues[ number ];
```

Comment:
Fix type so that it uses the values of the `features` object instead of its `keys`.